INSTALL

```diff
@@ -189,3 +189,7 @@ CONFIG+=g15-emulator (Mumble, Win32)
 CONFIG+=no-manual-plugin (Mumble)
  Don't include the built-in 'manual' positional
  audio plugin.
+
+CONFIG+=qtspeech (Mumble)
+ Use Qt's TTS system, part of the Qt Speech
```

Comment:
How about:

> Use Qt's text-to-speech system (part of the experimental QtSpeech module) instead of Mumble's own OS-specific text-to-speech implementations.

?